lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -121,7 +121,7 @@ pbr==5.4.2
 prettytable==0.7.2
 prov==1.5.1
 psutil==5.6.3
-pulsar-galaxy-lib==0.14.0.dev0
+pulsar-galaxy-lib==0.14.0.dev1
```

Comment:
Can you please sync the pin in the Pipfile?

Reply:
Done... I think.